src/map/utils/synthutils.cpp

```diff
@@ -361,9 +361,11 @@ namespace synthutils
                 chanceHQ = chanceHQ * 1.5f;
             }
 
+            float chanceHQMultiplier = settings::get<float>("map.CRAFT_HQ_CHANCE_MULTIPLIER"); // server configured additional HQ multiplier (default 1.0)
+
             // HQ success rate modifier.
             // See: https://www.bluegartr.com/threads/130586-CraftyMath-v2-Post-September-2017-Update page 3.
-            chanceHQ = chanceHQ + 100.0f * PChar->getMod(Mod::SYNTH_HQ_RATE) / 512.0f;
+            chanceHQ = (chanceHQ * chanceHQMultiplier) + 100.0f * PChar->getMod(Mod::SYNTH_HQ_RATE) / 512.0f;
```

Comment:
The setting should come last. Affecting both gear and base, etc...
So...
```c++
chanceHQ = chanceHQ + 100.0f * PChar->getMod(Mod::SYNTH_HQ_RATE) / 512.0f;
chanceHQ = chanceHQ * settings::get<float>("map.CRAFT_HQ_CHANCE_MULTIPLIER");
```
No need to define a variable for the setting if the setting is only used once.

Reply:
Refactored.